README.md

```diff
@@ -64,27 +61,43 @@ The Expunge Colorado Screener Tool backend uses the following tools and processe
     * The handler.js file accompanying the serverless.yml file is the lambda handler function (referenced in the functions property of serverless.yml)
 
-* AWS Lambda functions:
+* API gateway:
 
-    * AWS Lambda functions are event-driven functions that perform operations without having to manage/provision servers.
-    * One use of AWS Lambda is to process the HTTP requests from API gateway.
-        * The lambda function will listen for the appropriate httpMethod and update the AWS.DynamoDB.DocumentClient() object accordingly.
+    * AWS API gateway is being used as a web server to create HTTP endpoints for survey record storage (backend) and outcome page content retrieval (frontend).
+    * The reached HTTP endpoints upon survey completion will then trigger the associated Lambda functions through a direct integration in the AWS console.
 
-* API gateway:
+* AWS Lambda functions:
 
-    * AWS API gateway is being used as a web server to create HTTP endpoints (put, get) for survey completion.
-    * The reached HTTP endpoint will then trigger the associated Lambda function and update DynamoDB accordingly (through a direct integration option in AWS).
+    * AWS Lambda functions are event-driven functions that perform operations without having to manage/provision servers.
+    * This application uses AWS Lambda to process the HTTP requests from API gateway.
+        * For the backend survey record storage, the expunge lambda function will check the appropriate httpMethod and update DynamoDB accordingly.
+        * For the frontend content rendering, the contentful lambda function will then make an API call to the expunge contentful account and send the rich text content as JSON back to the front-end application.
 
 * DynamoDB:
 
     * DynamoDB allows for a simple document-based nosql database.
     * DynamoDB is being used for survey results storage;
     * Any new or updated survey will create a new entry in the db
 
+* Headless CMS
+
+    * A Headless CMS provides the ability for the content of a website or page to be created and stored in a location that does not directly update a website's UI, but instead can be accessed via an API to then render in customizable ways.
+    * This tool uses a Headless CMS for the survey outcome page in order to provide the Expunge Colorado team with the ability to customize this page easily.
+
+* contentful
+
+    * contentful was the Headless CMS service chosen for this project due to its low cost, ease of rich text formatting/updating for non-dev users and integration capabilities with react.
+    * Front-end administrative users from Expunge Colorado will sign into the Contentful platform to update the outcome page.
+    * This project has a Screener Outcomes space composed of content entries for each outcome.
+        * A *space* in contentful is a collection of content for achieving a particular goal. 
+        * An *entry* in contentful is a particular page of content with a Title and a Body.
```

Comment:
Just a note that an entry can contain any fields, which are specified by the content model. Our content model (Screener Outcome) has been configured with Title and Body

Reply:
ah gotcha! makes sense. added that info!